### PR TITLE
Add WSL Build package

### PR DESCRIPTION
**Pull request by existentialmutt**
- [X] I'm the package's author and/or maintainer.
- [X] I have have read [the docs][1].
- [X] I have tagged a release with a [semver][2] version number.
- [X] My package repo has a description and a README describing what it's for and how to use it.
- [X] My package doesn't add context menu entries.
- [X] My package doesn't add key bindings.

My package is WSL Build, an ExexCommand for creating build systems for projects running in WSL2

There are no packages like it in Package Control.

This is my first published Sublime Package so feedback is appreciated :)

Thank you!

### CODE REVIEW

File: repository/w.json [hunk 1]
```diff
@@ -1118,6 +1118,19 @@
 				}
 			]
 		},
+		{
+			"name": "WSL Build",
+			"details": "https://github.com/existentialmutt/wsl_build",
+			"description": "Create custom build systems for projects running in WSL2",
```
Comment (braver) [inline repository/w.json:1124]: This seems to match your GitHub repo description, so you can just remove it here (package control will find it in the repo).

Comment (braver) [reply repository/w.json:1124]: So, please remove that so we can wrap this up and ship your package!

**Comment by deathaxe** [comment]
1. The build command works only for Ubuntu distros and files located within WSL environment in its current state. 
2. It does also not opt-in to python 3.8, which each new plugin should do to be accepted.

_Filed issues about it at the repo._

Until those are fixed it is probably not worth adding to packagecontrol index.

**Comment by braver** [comment]
Please let us know if you want to re-open this PR and continue the review.

**Pull request closed without merging**